data-explorer/kusto/api/connection-strings/kusto.md

```diff
@@ -133,44 +133,64 @@ The following algorithm is generally used by clients for authentication against
 
 ## AAD-based authentication examples
 
-**AAD Federated authentication based-on the current logged-on user's identity (user will be prompted if required)**
+**AAD Federated authentication using the currently logged-on user identity (user will be prompted if required)**
 
 ```csharp
-// Option 1
-var serviceName = "help";
+var serviceNameAndRegion = "help";
 var authority = "contoso.com"; // Or the AAD tenant GUID: "..."
+
+// Recommended syntax
 var kustoConnectionStringBuilder = new KustoConnectionStringBuilder($"https://{serviceNameAndRegion}.kusto.windows.net")
```

Comment:
I would recommend we  treat the argument as a single value. Otherwise the "kusto.windows.net" here needs to be also parametrized. In fact, I would expect people to get this value from the properties displayed by the Azure portal.

Reply:
Fair point. I will update the PR with this recommendation